content/_data/changelogs/weekly.yml

```diff
@@ -2153,6 +2153,84 @@
     # pull: 3297 Jenkinsfile only
     # pull: 3303 too minor (i18n in setup wizard)
     # pull: 3281 too minor (fixes annotations)
+- version: "2.110"
+  date: 2018-02-05
+  changes:
+    - type: major rfe
+      issue: 22936
+      pull: 3289
+      message: >
+        It is no longer possible to rename jobs from their configuration page.
+        Jobs now have a link in the side panel titled "Rename" that links to a page specifically dedicated to renaming jobs.
+    - type: rfe
+      pull: 2926
+      issue: 43581
+      message: >
+        Show the "Add" button in lists of tool installations also on the top.
+    - type: rfe
+      pull: 3315
+      issue: 49788
+      message: >
+        Add <code>ConcurrentLinkedQueue</code> to white-listed classes for use in XStream (XML serialization) and Remoting (agent communication).
+    - type: rfe
+      pull: 3312
+      issue: 49795
+      message: >
+        Issue warnings to the system log when attempts are made to use classes with unpredictable names and serial forms (such as anonymous classes) in Remoting or XStream (de)serialization.
+    - type: bug
+      pull: 3313
+      issue: 49543
+      message: >
+        Make JEP-200 serialization whitelist more reliable on old versions of Tomcat.
+    - type: bug
+      pull: 3312
+      issue: 49795
+      message: >
+        Clean up the <code>build.xml</code> files of parameterized projects that contained unnecessary serialized data.
+    - type: bug
+      pull: 3307
+      message: >
+        Upgrade Winstone from 4.1.0 to 4.1.2 to prevent User session memory leak by setting the default idle session eviction timeout to 30 minutes.
+      references:
+        - issue: 49596
+        - url: https://github.com/jenkinsci/winstone/blob/master/CHANGELOG.md#412
+          title: full changelog
+    - type: bug
+      pull: 3316
+      issue: 49642
+      message: >
+        Make proxy views work inside folders.
+    - type: bug
+      pull: 3094
+      issue: 29470
+      message: >
+        Prevent <code>NullPointerException</code> in <code>AbstractProject#checkout</code> when the agent disconnects during a build.
+    - type: major rfe
+      pull: 3289
+      issue: 22936
+      message: >
+        Developer: Subclasses of <code>AbstractItem</code> can implement <code>AbstractItem#isNameEditable</code> and return true to get automatic support for renames.
+        Subclasses are also able to dynamically validate renames by implementing <code>AbstractItem#checkRename</code>.
+    - type: rfe
+      pull: 2926
+      message: >
+        Developer: Add support for also showing the "Add" button on top for <code>lib/form/repeatable</code> Jelly controls when the <code>enableTopButton</code> attribute is set to <code>true</code>.
+    - type: rfe
+      pull: 3277
+      message: >
+        Internal: Upgrade to PrototypeJS 1.7.3.
```

Comment:
"From 1.7.0 to 1.7.3". It would also make sense to add the changelog link: http://prototypejs.org/blog/